server/src/main/java/com/google/codeu/servlets/RestaurantServlet.java

```diff
@@ -22,22 +23,53 @@ public void init() {
     datastore = new Datastore();
   }
 
+  /** Function will return whether or not strNum is an Integer. */
+  public boolean isInt(String strNum) {
+    try {
+      int d = Integer.parseInt(strNum);
```

Comment:
You don't use the variable 'd'. You can just write

` return Integer.parseInt(strNum); `

and remove the true at the bottom.